web-ui/src/main/resources/catalog/components/viewer/owscontext/OwsContextService.js

```diff
@@ -687,8 +719,11 @@
        * @param {numeric} index of the layer in the tree
        */
       this.createLayer = function (layer, map, bgIdx, index, style) {
-        var server = layer.server[0];
-        var res = server.onlineResource[0];
+        var res = { href: "" };
+        if (layer.server) {
+          var server = layer.server[0];
+          res = server.onlineResource[0];
```

Comment:
I guess `server.onlineResource[0]` contains an object like `{ href: "XXXXX" }`?

Reply:
Usually yes, but it is a corner case in Sextant where the map context can define custom layer types without server.